File: app/src/main/java/jp/gr/java_conf/snake0394/loglook_android/storage/EquipmentFragmentPrefs.java
package jp.gr.java_conf.snake0394.loglook_android.storage;

import io.github.kobakei.spot.annotation.Pref;
import io.github.kobakei.spot.annotation.PrefField;

/**
 * {@link jp.gr.java_conf.snake0394.loglook_android.view.fragment.EquipmentFragment}
 */
@Pref(name = "pref_equip_fragment")
public class EquipmentFragmentPrefs {

    @PrefField(name = "sortType")
    public String sortType = "名前";

    @PrefField(name = "order")
    public String order = "昇順";

    public EquipmentFragmentPrefs() {
        //Empty constructor is needed
    }
}

File: app/src/main/java/jp/gr/java_conf/snake0394/loglook_android/storage/EquipmentFragmentPrefs.kt
package jp.gr.java_conf.snake0394.loglook_android.storage

import android.content.Context
import jp.takuji31.koreference.KoreferenceModel
import jp.takuji31.koreference.stringPreference

/**
 * [jp.gr.java_conf.snake0394.loglook_android.view.fragment.EquipmentFragment]
 */
class EquipmentFragmentPrefs(context: Context): KoreferenceModel(context = context, name = "pref_equip_fragment") {

    var sortType by stringPreference("名前")

    var order by stringPreference("昇順")
}//Empty constructor is needed